frontend/src/views/Updates.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+  <div class="bg-dark">
+    <div class="notification-header">
+      <span>{{$t("updates.notifications")}}</span>
+      <b-button id="markAllAsRead"
+        v-bind:readAll.sync="this.readAll"
+        @click="setAllAsRead">{{$t("updates.markAllAsRead")}}
+      </b-button>
+    </div>
+    <div v-for="update in updateNotifications" :key="update.hash">
+      <Update :hash="update.hash" :link="update.link" :title="update.title"
+        :timestamp="update.timestamp" v-bind:isRead.sync="update.isRead"
+        @refresh-update-popup="refreshUpdatePopup"/>
+    </div>
+  </div>
+</template>
+
+<script lang='ts'>
+import Vue, { PropType } from 'vue';
+import Update from '@/components/Update.vue';
+import { INotification } from '@/interfaces';
+
+export default Vue.extend({
+  props: {
+    notificationsFromAPI: {
+      type: Array as PropType<INotification[]>,
+      default() {
+        return [];
+      },
+    },
+    updateNotifications: {
+      type: Array as PropType<INotification[]>,
+      default() {
+        return [];
+      },
+    },
+    readAll: {
+      type: Boolean as PropType<boolean | undefined>,
+    },
+  },
+  methods: {
+    /**
+     * Mark all notifications as read
+     */
```

Comment:
Same in comments in this file, some give value, but some do not and need to be deleted.